invenio_vocabularies/cli.py

```diff
@@ -47,6 +51,51 @@ def _load_csv_data(path):
         return dicts
 
 
+def _create_languages_vocabulary(vocabulary_type_name, source_path):
+    """Load languages vocabulary."""
+    identity = Identity(1)
+    identity.provides.add(any_user)
+    service = VocabulariesService()
+
+    # Create vocabulary type
+    vocabulary_type = VocabularyType(name="languages")
+    db.session.add(vocabulary_type)
+    db.session.commit()
+
+    # Load data
+    records = []
+    instance_languages = [lang[0] for lang in current_i18n.get_languages()]
+
+    for language in pycountry.languages:
```

Comment:
minor: to harmonize and help understanding... avoid getting confused with:
  - language comes from pycountry
  - lang comes from instance configured languages
  
What about `pycountr_lang` and `instance_lang`?